lib/Exporter/Writer/MimeTypedWriterInterface.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Exporter\Writer;
+
+interface MimeTypedWriterInterface extends WriterInterface
+{
+    /**
+     * There can be several mime types for a given format, this method should
+     * return the most appopriate / popular one.
+     *
+     * @return string the mime type of the output
+     */
+    public function getMimeType();
+
+    /**
+     * Returns a string best describing the format of the output (the file
+     * extension is fine, for example).
+     *
+     * @return string a string without spaces, usable in a translation string
+     */
+    public function getFormat();
```

Comment:
Because the extension is not necessarily the format. For instance this method should return "yaml", not "yml", it's about describing the writer.